src/exec/execute_pipeline.c

```diff
@@ -0,0 +1,47 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   execute_pipeline.c                                 :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: reasuke <reasuke@student.42tokyo.jp>       +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/09/13 16:15:45 by reasuke           #+#    #+#             */
+/*   Updated: 2024/09/13 16:52:51 by reasuke          ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include <errno.h>
+#include <string.h>
+#include <unistd.h>
+
+#include "ast.h"
+#include "exec_internal.h"
+#include "utils.h"
+
+#define PIPE_IN 0
+#define PIPE_OUT 1
+
+void	execute_pipeline(t_ast *ast, char **envp, int fd_in, int fd_out)
```

Comment:
t_ast *variable name candidate: `node`

Reply:
good idea 👍🏼

Reply:
There's too many changes involved. So I'll handle this in #56. @alex216